baselines/dom.generated.d.ts

```diff
@@ -14454,8 +14454,7 @@ declare var SVGStringList: {
 };
 
 /** Corresponds to the SVG <style> element. */
-interface SVGStyleElement extends SVGElement {
-    disabled: boolean;
```

Comment:
should this be deleted? If it's not standard, or not widely implemented, then it's fine to delete, but if not, it would need to be re-added in addedTypes.json

Reply:
This website suggests it is not widely used: https://www.chromestatus.com/metrics/feature/timeline/popularity/. But I see it in this spec, https://developer.apple.com/documentation/webkitjs/svgstyleelement. So it will be added back.

Reply:
The apple documentation is not the spec. The right one does not has it: https://svgwg.org/svg2-draft/styling.html#InterfaceSVGStyleElement

Reply:
@saschanaz to make sure I understand, we just need to revert the last two commits, 6fa0fbc and f1f5a57, right?

Reply:
Exactly!